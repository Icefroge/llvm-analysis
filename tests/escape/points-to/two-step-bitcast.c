int a;
void f() {
  int * pa = &a;
  void * p = pa;
}
