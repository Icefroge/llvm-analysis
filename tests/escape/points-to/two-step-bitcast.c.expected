fromList [ ("%pa", fromList ["@a"])
         , ("%p", fromList ["@a"])
         ]
